packages/charting/src/components/StackedBarChart/StackedBarChart.styles.ts

```diff
@@ -60,6 +71,32 @@ export const getStyles = (props: IStackedBarChartStyleProps): IStackedBarChartSt
       fontSize: FontSizes.small,
       color: theme.palette.black,
       opacity: '0.6'
+    },
+    benchmarkContainer: {
+      position: 'relative',
+      height: '11.8px'
```

Comment:
Why aren't we using a whole number here? This will likely cause rendering weirdness across DPIs

Reply:
Change it to 12px.